### DISCUSSION TITLE
nrf_pairing

### DISCUSSION THREAD

**Discussion by Unicnight** [bitcraze/discussions#79] [Q&A]
i noticed there's a branch of [crazyflie2-nrf-firmware] named "nrf_pairing",can i download find it now 
thanks

**Comment by krichardsson** [comment]
Hi!

I'm not sure which branch you are referring to, I only see two branches that are not merged: `master` and `rzr`

https://github.com/bitcraze/crazyflie2-nrf-firmware/branches/all
